common/src/main/resources/assets/estrogen/lang/de_de.json

```
@@ -93,11 +155,45 @@
   "itemGroup.estrogen.estrogen": "Östrogen",
   "key.estrogen.dash": "Dash aktivieren",
   "recipe.entity_interaction.cant_be_baby": "Dieses Tier muss erwachsen sein!",
+  "subtitles.estrogen.cookie_jar.full": "Keksdose ist voll",
+  "subtitles.estrogen.cookie_jar.insert": "Keksdose wird eingefügt in",
   "subtitles.estrogen.dash": "Frauenpower-Dash",
+  "subtitles.estrogen.dream_block.enter": "Traumblock betreten",
+  "subtitles.estrogen.dream_block.exit": "Traumblock verlassen",
+  "subtitles.estrogen.entity.moth.death": "Rosarote Ahornmotte stirbt",
+  "subtitles.estrogen.entity.moth.fuzz_up": "Rosarote Ahornmotte pelzt",
+  "subtitles.estrogen.entity.moth.hurt": "Rosarote Ahornmotte verletzt sich",
+  "subtitles.estrogen.entity.moth.loop": "Rosarote Ahornmotte summt",
+  "subtitles.estrogen.music.amphitrite": "erora - Amphitrite",
+  "subtitles.estrogen.music.aurum_berry": "erora - Aurum Berry",
+  "subtitles.estrogen.music.g03c": "erora - G03C",
+  "subtitles.estrogen.music.inferred_dreams": "erora - Inferred Dreams",
+  "subtitles.estrogen.music.inner_selfrealization": "erora - Inner Selfrealization",
+  "subtitles.estrogen.music.sleeping_dreams": "erora - Sleeping Dreams",
+  "subtitles.estrogen.music.trust_yourself": "erora - Trust Yourself",
   "tag.fluid.estrogen.urine": "Urin",
-  "tag.item.curios.thighs": "Oberschenkel",
-  "tag.item.estrogen.copper_plates": "Kupferplatten",
+  "tag.item.c.cookies": "Kekse",
+  "tag.item.c.leather_items": "Leder Item",
+  "tag.item.c.light_emitters": "Licht Emitter",
+  "tag.item.curios.thighs": "Oberschenkelstrümpfe",
+  "tag.item.estrogen.chest_armor_ignore": "Brustpanzerung Ignorieren",
   "tag.item.estrogen.uwufying": "UwUifizierend",
   "tag.item.trinkets.legs.thighs": "Oberschenkel",
-  "trinkets.slot.legs.thighs": "Oberschenkel"
+  "tooltip.thigh_highs.estrogen.agender": "§8A§7g§fe§an§fd§7e§8r",
+  "tooltip.thigh_highs.estrogen.aromantic": "§aAro§fman§8tic",
+  "tooltip.thigh_highs.estrogen.asexual": "§5As§fex§8ual",
+  "tooltip.thigh_highs.estrogen.bisexual": "§dBis§5ex§1ual",
+  "tooltip.thigh_highs.estrogen.demisexual": "De§8mi§fse§5xu§fal",
+  "tooltip.thigh_highs.estrogen.genderfluid": "§dGe§fnd§5er§8fl§9uid",
+  "tooltip.thigh_highs.estrogen.genderqueer": "§aGen§fder§5queer",
+  "tooltip.thigh_highs.estrogen.intersex": "§eInt§5er§esex",
+  "tooltip.thigh_highs.estrogen.lesbian": "§cL§6es§fb§dia§5n",
+  "tooltip.thigh_highs.estrogen.non_binary": "§eNon§f-Bi§5na§8ry",
+  "tooltip.thigh_highs.estrogen.pansexual": "§dPan§esex§bual",
+  "tooltip.thigh_highs.estrogen.polyamorous": "§bPoly§fam§coro§8us",
+  "tooltip.thigh_highs.estrogen.polysexual": "§dPoly§asex§bual",
+  "tooltip.thigh_highs.estrogen.pride": "§cP§6r§ei§ad§5e",
+  "tooltip.thigh_highs.estrogen.trans": "§bTr§dan§fsge§dnd§ber",
+  "tooltip.thigh_highs.estrogen.vincian": "§3Vi§an§fc§bia§9n",
+  "trinkets.slot.legs.thighs": "Oberschenkelstrümpfe"
```

```suggestion
  "trinkets.slot.legs.thighs": "Oberschenkel"
```
Same as before, the slot should probably stay as "Oberschenkel" for the literal translation "thighs".